codeSnippets/snippets/autoreload/README.md

```diff
@@ -0,0 +1,19 @@
+# Auto-reload
+
+A sample Ktor project showing [Auto-reload](https://ktor.io/docs/auto-reload.html) functionality. This feature works only in a development mode, which is enabled in [application.conf](resources/application.conf).
```

Comment:
development mode also could be enabled by `-ea` VM option

Reply:
in this example, it's enabled in application.conf. Other ways are shown in a referenced topic.

Reply:
Yes, but you are describing feature here in general:
> This feature works only in a development mode, which is enabled in [application.conf](resources/application.conf)

Reply:
Oh I see the `application.conf` is the concrete sample file